types/index.d.ts

```diff
@@ -31,6 +31,7 @@ declare namespace fastifyHttpProxy {
     proxyPayloads?: boolean;
     preHandler?: preHandlerHookHandler;
     beforeHandler?: preHandlerHookHandler;
+    preValidation?: preValidationHookHandler;
```

Comment:
Can you add a test for this type in https://github.com/fastify/fastify-http-proxy/blob/master/types/index.test-d.ts?

Reply:
Addressed here: 0c0e505c7180d3f31d51e2c081d334ba7da580d5